src/functions/arrow.js

```diff
@@ -119,9 +129,15 @@ defineFunction({
             }
         } else if (group.below) {
             lowerNode = mml.buildGroup(group.below, options);
+            lowerNode = paddedNode(lowerNode);
             node = new mathMLTree.MathNode("munder", [arrowNode, lowerNode]);
         } else {
-            node = new mathMLTree.MathNode("mover", [arrowNode]);
+            // This should never happen.
+            // Parser.js throws an error if there is no argument.
+            node = new mathMLTree.MathNode("mpadded", []);
+            node.setAttribute("width", "+0.6em");
+            node.setAttribute("lspace", "0.3em");
+            node = new mathMLTree.MathNode("mover", [arrowNode, node]);
```

Comment:
```js
node = paddedNode();
```

Reply:
Done, with a modification as needed to satisfy `flow`.